### ISSUE TITLE
Allow for variables to be fed in instead of using OS environment variables

### ISSUE THREAD

**Issue by Electro707** [peeter123/digikey-api#14]
As the title says, this library should allow for some information like `client-id` to be fed into it with the user application instead of using environment variables. Then these variables can be stored in the library's configuration file. The configuration file as well should be given to the library as an argument instead of using an environment variable.

**Comment by Electro707** [comment]
Started work on this in my branch: https://github.com/Electro707/digikey-api/tree/json_config_storage
So far for V3 it's working. I don't plan on updating the V2 code to handle this tough.

**Comment by eeintech** [comment]
I would like to see this too :smiley: 
Can you create a PR?

**Comment by Electro707** [comment]
Yup, created under #15. Set as a WIP as more things needs to be done before it can be merged upstream, some things up for discussion.
